util/test/test_query_ve.py

```diff
@@ -0,0 +1,89 @@
+import time
```

Comment:
In this file, move test_populateSubgraph() into test_query.py, as a _test_populateSubgraph(). Why: avoids an extra 30 s of loops, and the code for loops you needed to add; instead it reuses the infrastructure already set up in test_query.py.